app/tests/spec/models/user.js

```diff
@@ -674,62 +674,80 @@ define(function (require, exports, module) {
         });
       });
 
-      describe('with an unverified account', function () {
-        beforeEach(function () {
-          account = user.initAccount({ email: 'email', uid: 'uid' });
+      // Ensure that content-server can correctly handle any emailSent response
+      var emailSentTestCases = [{
```

Comment:
Can you use `const` instead of `var` here? We're living in the (mostly) present in the content server!